examples/16.add_remove_subaccount.py

```diff
@@ -0,0 +1,37 @@
+from config import TEST_ACCT_KEY, TEST_SUB_ACCT_KEY, TEST_NETWORK
+from bluefin_v2_client import (
+    BluefinClient,
+    Networks,
+)
+import asyncio
+
+
+async def main():
+  # initialize the parent account client
+  clientParent = BluefinClient(True, Networks[TEST_NETWORK], TEST_ACCT_KEY)
+  await clientParent.init(True)
+
+  # initialize the child account client
+  clientChild = BluefinClient(True, Networks[TEST_NETWORK], TEST_SUB_ACCT_KEY)
+  await clientChild.init(True)
+
+  print("Parent: ", clientParent.get_public_address())
+  print("Child: ", clientChild.get_public_address())
+
+  # add child account as subaccount
+  status = await clientParent.update_sub_account(
+      clientChild.get_public_address(), True
+    )
+  print("Sub account added: {}\n".format(status))
+
+  # remove child account as subaccount
+  status = await clientParent.update_sub_account(
+      clientChild.get_public_address(), True
```

Comment:
I think it should be false?

Reply:
Good catch, updating it here and in documentation